### PR TITLE
v3

### PR DESCRIPTION
**Pull request by tgabi333**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+LocalMedia
+==========
+
+- 3.0.0
+    - [BREAKING] removed screenSharingSupport flag
```
Comment (fippo) [inline CHANGELOG.md]: oh... that one is hard to provide reliably so +1 to removing it.

**Comment by fippo** [comment]
that looks pretty awesome!

**Comment by tgabi333** [comment]
@fippo Could you please release it along with getUserMedia and traceablepeerconnection and RTCPeerConnection if you have time?

**Comment by xdumaine** [comment]
@tgabi333 I published localmedia@3.0.0

**Comment by xdumaine** [comment]
Nice work on this btw. Thanks for the changelog!

**Comment by tgabi333** [comment]
Thank you @xdumaine , could you please release minors in: https://github.com/otalk/traceablepeerconnection and https://github.com/otalk/RTCPeerConnection and https://github.com/otalk/getUserMedia/

**Comment by xdumaine** [comment]
I did getScreenMedia and RTCPeerConnection, but I don't have npm publish rights to traceablepeerconnection or getUserMedia. @fippo @legastero Thoughts on adding myself and @tgabi333 to those?

**Comment by tgabi333** [comment]
Please @fippo, @legastero, would you be so kind to release npm packages for the following repos:
- otalk/getScreenMedia
- otalk/traceablepeerconnection
- otalk/getUserMedia

It could help us to use webrtc/adapter v2.

Thanks in advance.

**Pull request merged**